### PR TITLE
Integrals of parametric functions

### PR DESCRIPTION
**Pull request by jzwar**
# Overview
Integrals on embedded splines using 
```math
\mathcal{J}^S = \big(\text{det}(\mathbf{J}^T\mathbf{J}) \big)^{\frac{1}{2}}
```

Baseline for @j042 

## Checklists
* [ ] Documentations are up-to-date.
* [ ] Added example(s)
* [ ] Added test(s)

### CODE REVIEW

**Comment by j042** [comment]
Thanks! I guess the last commit is only related?

**Comment by jzwar** [comment]
Yes, the others are related to ms

File: splinepy/helpme/integrate.py [hunk 1]
```diff
@@ -135,3 +246,11 @@ def __init__(self, spl):
...
     def volume(self, *args, **kwargs):
         return volume(self._helpee, *args, **kwargs)
+
+    def parametric_function(self, *args, **kwargs):
+        return parametric_function(self._helpee, *args, **kwargs)
+
+
+# Use function docstrings in Extractor functions
+Integrator.volume.__doc__ = volume.__doc__
+Integrator.parametric_function.__doc__ = parametric_function.__doc__
```
Comment (j042) [inline splinepy/helpme/integrate.py:256]: @j042 update this

**Review by j042 — APPROVED** [review]
Very nice, thank you!

**Pull request merged**